src/jit/CMakeLists.txt

```diff
@@ -168,10 +168,15 @@ endif()
 
 set(CLR_EXPORTED_SYMBOL_FILE ${CLRJIT_EXPORTS_DEF})
 
+set(JIT_BASE_NAME clrjit)
+if (CLR_BUILD_JIT32)
+  set(JIT_BASE_NAME ryujit)
+endif()
```

Comment:
If JIT32 is being built, it gets to be named `clrjit`. The change here avoids a collision between the two. Whichever JIT is named `clrjit` will be linked into the runtime and crossgen and distributed as a package.